src/lsp/main.ts

```diff
@@ -430,15 +430,20 @@ async function startClient(
     lspStatus.text = '$(rocket) clojure-lsp';
     lspStatus.show();
     client.start();
-    await onReadyPromise;
+    await onReadyPromise.catch((e) => {
+        console.error('clojure-lsp:', e);
+        setStateValue(LSP_CLIENT_KEY_ERROR, e);
+    });
```

Comment:
it's funny, before I worked on this project I didn't know you could mix `await` + `.catch`

Reply:
I didn't know this either 😄 .